services/statestorage/test/_stress_test.go

```diff
@@ -0,0 +1,79 @@
+package test
```

Comment:
Since this is a manual test, I prefer the following filename convention which we already use elsewhere in the system:

`services/statestorage/test/_manual/stress_test.go`

[Example](https://github.com/orbs-network/orbs-network-go/tree/master/test/_manual)

and add a small README.md next to it in the folder explaining how to run it and why

Reply:
moved file and added README.md